### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by FlyersWeb** [kmkzt/react-hooks-svgdrawing#86]
Hi,

I'm trying to use your module in a project of mine based on NextJS@9.5.2 and TypeScript@3.9.7 but when I compile it I get the following error :

```
> Build error occurred
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/flyers/Lab/webapp/node_modules/react-hooks-svgdrawing/lib/index.cjs.js
require() of ES modules is not supported.
require() of /Users/flyers/Lab/webapp/node_modules/react-hooks-svgdrawing/lib/index.cjs.js from /Users/flyers/Lab/webapp/.next/server/pages/music/AnnotateSheet.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.cjs.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/flyers/Lab/webapp/node_modules/react-hooks-svgdrawing/package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1172:13)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.QVvi (/Users/flyers/Lab/webapp/.next/server/pages/music/AnnotateSheet.js:202:18)
    at __webpack_require__ (/Users/flyers/Lab/webapp/.next/server/pages/music/AnnotateSheet.js:23:31)
    at Module.qbna (/Users/flyers/Lab/webapp/.next/server/pages/music/AnnotateSheet.js:423:40)
    at __webpack_require__ (/Users/flyers/Lab/webapp/.next/server/pages/music/AnnotateSheet.js:23:31)
    at Object.28 (/Users/flyers/Lab/webapp/.next/server/pages/music/AnnotateSheet.js:99:18) {
  type: 'NodeError',
  code: 'ERR_REQUIRE_ESM'
```

Do you know how can I handle this error ?

Best regards.

**Comment by kmkzt** [comment]
@FlyersWeb 
I'm sorry to reply late.

I am in the process of migrating to [this repository](https://github.com/kmkzt/svg-drawing/tree/master/packages/react), but the [example code](https://github.com/kmkzt/svg-drawing/tree/master/examples/docs) is working, so I think it will work here.

Is there any code to reproduce?

**Comment by FlyersWeb** [comment]
Sadly I can't share any code but I resolved the issue by removing the `"type": "module"` from the `package.json` 
If anyone has the same issue some day...

**Comment by kmkzt** [comment]
@FlyersWeb 
Sorry. Fixed in v2.1.2.
